localization/english/wc_laws_l_english.yml

```diff
@@ -0,0 +1,27 @@
+﻿l_english:
+ dark_authority:0 "[dark_authority|E]"
+ dark_authority_0:0 "Disparate Cults"
```

Comment:
![image](https://user-images.githubusercontent.com/46576860/126080055-26273da7-c5bf-49e6-bebb-80ad50f70550.png)

Are you sure it's the right word? Maybe you wanted to use "disperse" instead?

```suggestion
 dark_authority_0:0 "Disperse Cults"
```

But I would use this one instead
```suggestion
 dark_authority_0:0 "Scattered Cults"
```

Reply:
I like the Scattered Cults best of these alternatives :)

Reply:
> Are you sure it's the right word?

Yes.

Reply:
![image](https://user-images.githubusercontent.com/46384992/126084058-74153d45-b1f4-4e27-a286-18e6a8e78a16.png)

Reply:
> Yes.

Well, it may sound good for tribes, but it doesn't work for cults.